packages-exp/auth-exp/src/core/auth/auth_impl.test.ts

```diff
@@ -117,6 +118,102 @@ describe('AuthImpl', () => {
       );
     });
   });
+
+  describe('change listeners', () => {
+    // // Helpers to convert auth state change results to promise
+    // function onAuthStateChange(callback: NextFn<User|null>)
+
+    it('immediately calls authStateChange if initialization finished', done => {
+      const user = testUser('uid');
+      auth.currentUser = user;
+      auth._isInitialized = true;
+      auth.onAuthStateChanged(user => {
+        expect(user).to.eq(user);
```

Comment:
can you use  expect(callback).to.have.been.calledWith() instead?

Reply:
Done

Reply:
oh, maybe I misunderstood, it looks like you used calledWith() down below, but not in these test cases